tests/phpunit/tests/test-wpml-gutenberg-config-option.php

```diff
@@ -14,33 +14,163 @@ class Test_WPML_Gutenberg_Integration_Config_Option extends OTGS_TestCase {
 	public function it_updates_option_from_config() {
 		$subject = new WPML_Gutenberg_Config_Option();
 
-		$block_type = 'block/type';
-		$block_xpath = array( array( 'xpath1' ), array( 'xpath2' ) );
-		$expected_block_path = array( 'xpath1', 'xpath2' );
+		$block_type            = 'block/type';
+		$block_xpath           = array( array( 'xpath1' ), array( 'xpath2' ) );
+		$expected_block_config = array(
+			'xpath' => array( 'xpath1', 'xpath2' ),
+		);
 
 		$block_data = array(
-			'attr' => array( 'type' => $block_type, 'translate' => '1' ),
+			'attr'  => array( 'type' => $block_type, 'translate' => '1' ),
 			'xpath' => $block_xpath,
 		);
 
 		$config_settings = array(
 			'wpml-config' => array(
 				'gutenberg-blocks' => array(
 					'gutenberg-block' => array( $block_data ),
-				)
-			)
+				),
+			),
 		);
 
 		\WP_Mock::userFunction( 'update_option',
-			array(
-				'times' => 1,
-				'args' => array( WPML_Gutenberg_Config_Option::OPTION, array( $block_type => $expected_block_path ) )
-			)
+		                        array(
+			                        'times' => 1,
+			                        'args'  => array(
+				                        WPML_Gutenberg_Config_Option::OPTION,
+				                        array( $block_type => $expected_block_config ),
+			                        ),
+		                        ) );
+
+		$subject->update_from_config( $config_settings );
```

Comment:
@strategio maybe I'm just missing it, but where is `$config_settings` defined?

Reply:
It's defined here https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/pull/39/files#diff-df6372bc952d21fce216cca2189ba47cR28 @andreasciamanna. The display of the changes is not really helping.

Reply:
👍